### PR TITLE
Create PBAB-QUICK-START.md

### PR DESCRIPTION
**Pull request by Jbern16**

### CODE REVIEW

File: PBAB-QUICK-START.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+1. Discord - https://discord.com/developers/docs/resources/webhook
+2. Twitter - https://developer.twitter.com/en/docs/twitter-api
+
+# Fork 
+
+Fork this repo into your organization.
+
+# Adjust Code. 
+
+1. Allow your contracts here - https://github.com/ArtBlocks/alertbot/blob/main/src/index.ts#L13-L15, remove ab contracts.
```
Comment (jakerockland) [inline PBAB-QUICK-START.md:12]: Nit: ", remove ab contracts" --> " (and remove the Art Blocks contracts)"

File: PBAB-QUICK-START.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Fork this repo into your organization.
+
+# Adjust Code. 
+
+1. Allow your contracts here - https://github.com/ArtBlocks/alertbot/blob/main/src/index.ts#L13-L15, remove ab contracts. 
+2. Update this endpoint https://github.com/ArtBlocks/alertbot/blob/main/src/api_data.ts#L46 - use {contractAddress/tokenId} 
+3. Provide location for smaller images. If none available, use your defualt image bucket. https://github.com/ArtBlocks/alertbot/blob/main/src/config.ts#L20. 
+      - NOTE: If default image bucket is used, you may run up against twitter image size limits.
+5. Set rest of environment https://github.com/ArtBlocks/alertbot/blob/main/.env.example. When ready, we can provide the webhook secret.
```
Comment (jakerockland) [inline PBAB-QUICK-START.md:16]: @Jbern16 Is this webhook secret something that we can issue on a per-partner basis (and revoke if needed due to access breech or something like that)?

File: PBAB-QUICK-START.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+# Deploy.
+
+We like using Heroku as it is easy to deploy quickly. If you rather not use heroku, just keep in mind that a redis instance is needed for this to work. 
+
+1. Follow this documentation - https://devcenter.heroku.com/articles/git
+2. Provision REDIS - https://devcenter.heroku.com/articles/heroku-redis (use hobby level, so you can skip the SSL setup)
+3. Set Environment Variables in Heroku console. 
+
```
Comment (jakerockland) [inline PBAB-QUICK-START.md:26]: Would add one more section at the bottom that says "Having trouble?" and then asks that folks file a bug here in this repo with a link to file a new issue (and perhaps add a template for issue filing in this repo :) )?

File: PBAB-QUICK-START.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Gain Access to Twitter and Discord APIs
```
Comment (jakerockland) [inline PBAB-QUICK-START.md:1]: Let's link out to this new `.md` file in https://github.com/ArtBlocks/alertbot#supporting-your-pbab-project

**Review by jakerockland — APPROVED** [review]
LGTM assuming updates to address the feedback here. Great stuff, thank you @Jbern16

File: PBAB-QUICK-START.md (hunk 2 above)
Comment (Jbern16) [reply PBAB-QUICK-START.md:16]: Will have to sync with @yoshiwarab to see best way to go about this

File: PBAB-QUICK-START.md (hunk 3 above)
Comment (Jbern16) [reply PBAB-QUICK-START.md:26]: currently don't have access to configure a template, will file a ticket to track this

File: PBAB-QUICK-START.md (hunk 2 above)
Comment (Jbern16) [reply PBAB-QUICK-START.md:16]: That being said, I think we can figure this out when a partner gets to this point and shouldn't hold up oboarding interested parties. 

@yoshiwarab thoughts?

Comment (jakerockland) [reply PBAB-QUICK-START.md:16]: > That being said, I think we can figure this out when a partner gets to this point and shouldn't hold up oboarding interested parties.

Yep yep, I agree, doesn't need to be a blocker for merging this and starting to help interested partners get moving on onboarding. That being said PLEASE PLEASE PLEASE lets not share a webhook secret that is universal across all partners, especially if it is also shared with us as a partner.

**Review by jakerockland — APPROVED** [review]
Woohoo, thank you. LGTM! :)

File: PBAB-QUICK-START.md (hunk 2 above)
Comment (Jbern16) [reply PBAB-QUICK-START.md:16]: yes of course

**Pull request merged**